docs/reference/mapping.asciidoc

```diff
@@ -123,44 +123,159 @@ You can create field mappings when you
 fields to an existing index with the <<indices-put-mapping,PUT mapping API>>.
 
 [float]
-== Updating existing field mappings
+[[create-mapping]]
```

Comment:
In the section above on "Explicit Mappings", we reference external API pages for 'create index' and 'put mapping'. Do we instead want to link to the sections below since it's nice to stay within a one page while reading up on a topic?

Reply:
Great idea! Added with cbe9827.